testcases/OpTestFlash.py

```diff
@@ -226,10 +226,10 @@ def runTest(self):
                 print "Backup of skiroot kernel lid is in %s/80f00101_bkp.lid" % self.ext_lid_test_path
                 self.cv_BMC.fsp_run_command("cp -f /opt/extucode/80f00102.lid %s/80f00102_bkp.lid" % self.ext_lid_test_path)
                 print "Backup of skiroot initrd lid is in %s/80f00102_bkp.lid" % self.ext_lid_test_path
+                self.cv_BMC.fsp_run_command("rm -f /opt/extucode/80f00101.lid")
+                self.cv_BMC.fsp_run_command("rm -f /opt/extucode/80f00102.lid")
                 self.scp_file(self.skiroot_kernel, self.ext_lid_test_path + "/80f00101.lid")
                 self.scp_file(self.skiroot_initramfs, self.ext_lid_test_path + "/80f00102.lid")
-                self.cv_BMC.fsp_run_command("cp -f %s/80f00101.lid /opt/extucode/" % self.ext_lid_test_path)
-                self.cv_BMC.fsp_run_command("cp -f %s/80f00102.lid /opt/extucode/" % self.ext_lid_test_path)
```

Comment:
You have to fix 80f00101.lid and 80f00102.lid properly. Just deleting `cp` is not enough. You have to change `scp` as well.

Also by any chance did you figureout what's wrong with existing code (I did enable few log.. but no luck:-( ).

-Vasant

Reply:
ah sorry, forget that changing scp for skiroot lids.  Will change that.
not able to figure out why it's not working.